spec/models/manageiq/providers/google/cloud_manager/refresher_spec.rb

```diff
@@ -41,24 +41,24 @@ def assert_table_counts
     expect(ExtManagementSystem.count).to eql(1)
     expect(Flavor.count).to              eql(18)
     expect(AvailabilityZone.count).to    eql(13)
-    expect(VmOrTemplate.count).to        eql(347)
+    expect(VmOrTemplate.count).to        eql(348)
     expect(Vm.count).to                  eql(2)
-    expect(MiqTemplate.count).to         eql(345)
-    expect(Disk.count).to                eql(0)
+    expect(MiqTemplate.count).to         eql(346)
```

Comment:
We save inventory with VCR so at the time the cassette was recorded the first time there were 345 templates.  I had to re-record the cassette to pull back the disk inventory and in that time another template was added.